src/senselab/audio/tasks/data_augmentation.py

```diff
@@ -1,44 +1,92 @@
 """This module implements some utilities for audio data augmentation."""
 
-from typing import Any, Dict
+from typing import Any, Dict, List, Union
 
 import torch
 from datasets import Dataset
 from torch_audiomentations import Compose
 
+from senselab.utils.data_structures.audio import (
+    Audio,
+    batch_audios,
+    unbatch_audios,
+)
+from senselab.utils.device import DeviceType, _select_device_and_dtype
 from senselab.utils.tasks.input_output import (
     _from_dict_to_hf_dataset,
     _from_hf_dataset_to_dict,
 )
 
 
-def augment_hf_dataset(
-    dataset: Dict[str, Any], augmentation: Compose, audio_column: str = "audio"
-) -> Dict[str, Any]:
+def augment_audio_dataset(
```

Comment:
Can we say "audios" instead of "audio_dataset" since it's a list of audios for how it is rn?

Reply:
Also, not urgent, for the very good documentation that we will write, it's a good idea to include some suggestions for parameters to use for the different types of augmentations